js/utils/templateHelpers.js

```diff
@@ -1,10 +1,6 @@
 import app from '../app';
 
-const polyT = function (...args) {
-  return app.polyglot.t(...args);
-};
-
-export {
-  polyT,
-};
+export function polyT(...args) {
+  return app.polyglot.t(args);
```

Comment:
I think you need the ellipses for the args your passing to app.polyglot.t as well.